jenkins/jenkins.go

```diff
@@ -0,0 +1,173 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package jenkins
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+const jenkinsURL = "http://192.168.1.10:8080"
+const jenkinsJobName = "gui_kbfs_dokan_1"
+
+func StartBuild(ClientRev string, KBFSRev string, JsonUpdateFilename string) (string, error) {
+
+	buildurl := jenkinsURL + "/job/" + jenkinsJobName + "/buildWithParameters"
+	var params string
+	if len(ClientRev) > 0 {
```

Comment:
"Both are used within Go's standard libraries... Both are idiomatic and and clear enough. It is more a matter of personal taste and about clarity."
http://stackoverflow.com/questions/18594330/the-best-way-to-test-for-an-empty-string-in-go?answertab=votes#tab-top